app/Providers/RouteServiceProvider.php

```diff
@@ -28,7 +28,9 @@ public function boot(Router $router)
     {
         parent::boot($router);
 
-        $router->model('conference', Conference::class);
+        $router->bind('conference', function ($conference) {
```

Comment:
I'll merge this but there is a simpler solution we should change to later: you can teach an eloquent model which column is its URL resolver.